src/commands/createFunction/durableSteps/ConfigureDurableProjectStep.ts

```diff
@@ -0,0 +1,127 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, AzureWizardExecuteStep, IParsedError, parseError } from '@microsoft/vscode-azext-utils';
+import * as path from "path";
+import { Progress } from 'vscode';
+import { ConnectionKey, DurableBackend, hostFileName, ProjectLanguage } from '../../../constants';
+import { ext } from '../../../extensionVariables';
+import { IHostJsonV2 } from '../../../funcConfig/host';
+import { MismatchBehavior, setLocalAppSetting } from '../../../funcConfig/local.settings';
+import { localize } from '../../../localize';
+import { cpUtils } from '../../../utils/cpUtils';
+import { durableUtils, netheriteUtils, sqlUtils } from '../../../utils/durableUtils';
+import { pythonUtils } from '../../../utils/pythonUtils';
+import { venvUtils } from '../../../utils/venvUtils';
+import { IFunctionWizardContext } from '../IFunctionWizardContext';
+
+export class ConfigureDurableProjectStep<T extends IFunctionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 225;
+
+    public async execute(context: T, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        const configuring: string = localize('configuringDurableProject', 'Configuring durable project settings...');
+        progress.report({ message: configuring });
+
+        await this._configureHostAndLocalSettingsJson(context);
+        await this._installDurableDependencies(context);
+    }
+
+    public shouldExecute(context: T): boolean {
+        return !!context.newDurableStorageType;
+    }
+
+    private async _configureHostAndLocalSettingsJson(context: T): Promise<void> {
```

Comment:
I'm not sure about this, but wanted to bring it up:

Rather than failing the entire `createFunction` wizard, would it be better to just output the error at the end, but continue creating anyway?

I feel like this will block users in a way that they may not be able to fix easily. Like, maybe the path is janky or we don't have access to it, ect.

But if we just alert them to the error, they can manually resolve it themselves.

Reply:
Yeah we could do it that way.  I think if we go this route, I should also go into `NetheriteConfigureHostStep` and make sure it won't error when the host is missing then.  I originally wasn't trying to prevent this for the same reason as why I was throwing the error in this configure step.